### PR TITLE
Eureka Fix

### PR DESCRIPTION
**Pull request by minesettimi**
A number was incorrect on the eureka fix that allowed for the engineer to still teleport to home with the command.

### CODE REVIEW

**Comment by minesettimi** [comment]
fixed number mixup now fully tested and working

File: addons/sourcemod/scripting/freak_fortress_2/commands.sp [hunk 1]
```diff
@@ -304,15 +304,18 @@ public Action Command_EurekaTeleport(int client, const char[] command, int args)
...
 		int entity = MaxClients + 1;
 		while((entity = FindEntityByClassname(entity, "obj_teleporter")) != -1)
 		{
 			if(GetEntPropEnt(entity, Prop_Send, "m_hBuilder") == client && GetEntProp(entity, Prop_Send, "m_iObjectMode") == view_as<int>(TFObjectMode_Exit))
 			{
 				if(!GetEntProp(entity, Prop_Send, "m_bBuilding"))
 				{
-					char buffer[4];
-					GetCmdArg(1, buffer, sizeof(buffer));
 					if(StringToInt(buffer) == 1)
```
Comment (Batfoxkid) [inline addons/sourcemod/scripting/freak_fortress_2/commands.sp:319]: You can remove this check here since it's already checked for in the added lines.

**Review by Batfoxkid — CHANGES_REQUESTED** [review]

**Comment by minesettimi** [comment]
was fixed in the 2nd commit

**Comment by Batfoxkid** [comment]
> was fixed in the 2nd commit

[`if(StringToInt(buffer) == 1)`](https://github.com/Batfoxkid/Freak-Fortress-2-Rewrite/pull/24/files#diff-b43c26b85602304d4197539cedc39d08fe0967515bffa2fff3f81bbd245d7ad6R309) still exists even with [`if (StringToInt(buffer) == 0)`](https://github.com/Batfoxkid/Freak-Fortress-2-Rewrite/pull/24/files#diff-b43c26b85602304d4197539cedc39d08fe0967515bffa2fff3f81bbd245d7ad6R319) being used before.

**Comment by minesettimi** [comment]
should be good now

File: addons/sourcemod/scripting/freak_fortress_2/commands.sp [hunk 2]
```diff
@@ -304,16 +304,20 @@ public Action Command_EurekaTeleport(int client, const char[] command, int args)
...
 		while((entity = FindEntityByClassname(entity, "obj_teleporter")) != -1)
 		{
 			if(GetEntPropEnt(entity, Prop_Send, "m_hBuilder") == client && GetEntProp(entity, Prop_Send, "m_iObjectMode") == view_as<int>(TFObjectMode_Exit))
 			{
 				if(!GetEntProp(entity, Prop_Send, "m_bBuilding"))
 				{
-					char buffer[4];
-					GetCmdArg(1, buffer, sizeof(buffer));
-					if(StringToInt(buffer) == 1)
+					if(teletype == 1)
```
Comment (Batfoxkid) [inline addons/sourcemod/scripting/freak_fortress_2/commands.sp:320]: This check still isn't needed at you already checked if the teletype isn't zero in the above change.

**Comment by minesettimi** [comment]
adjusted the code to only check the type of teleport once

**Review by Batfoxkid — APPROVED** [review]

**Pull request merged**